modules/openapi-generator/src/main/resources/swift5/api.mustache

```diff
@@ -8,7 +8,10 @@
 import Foundation{{#usePromiseKit}}
 import PromiseKit{{/usePromiseKit}}{{#useRxSwift}}
 import RxSwift{{/useRxSwift}}{{#useCombine}}
-import Combine{{/useCombine}}{{#swiftUseApiNamespace}}
+#if canImport(Combine)
+import Combine
+#endif
+{{/useCombine}}{{#swiftUseApiNamespace}}
```

Comment:
Could you please put `#endif{{/useCombine}}{{#swiftUseApiNamespace}}` in the same line to avoid generating an extra line?

Reply:
Jep :-)